types/Yargs.ts

```diff
@@ -40,12 +40,19 @@ export type MigrateAppOptions = CommonArgs &
   AccountArgs &
   EnvironmentArgs &
   ConfigArgs & {
-    name: string;
-    dest: string;
-    appId: number;
+    name?: string;
+    dest?: string;
+    appId?: number;
     platformVersion: string;
   };
 
+export type ProjectMigrateOptions = CommonArgs &
```

Comment:
These command specific types can live in the command file. Also this is a big nit, but for consistency we're using "args" instead of options now

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1424/commits/bae540f2e4f2a36ff96f6a4e276d0150841e23e6